vNext/AISKU/Tests/applicationinsights.e2e.tests.ts

```diff
@@ -372,14 +377,32 @@ export class ApplicationInsightsTests extends TestClass {
                 stepDelay: 5000,
                 steps: [
                     () => {
-                        fetch('https://httpbin.org/status/200', { method: 'GET' });
+                        this._ai["dependencies"]["_config"].disableCorrelationHeaders = true;
+                        this._appId = this._ai["dependencies"]["_config"].appId;
+                        this._ai["dependencies"]["_config"].appId = null;
```

Comment:
Why was this and the `disableCorrelationHeaders` change needed? Consider making this and resetting back to the old values its own step